src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -611,15 +647,15 @@ private static bool IsMarkedToSkipConversion(PyObject pyObj)
         ///     Emitted immediately after execution ends or fails
         /// </summary>
         [SupressImportIntoVM]
-        public static event Dynamo.PythonServices.EvaluationFinishedEventHandler EvaluationFinished;
+        public override event EvaluationFinishedEventHandler EvaluationFinished;
 
         /// <summary>
         /// Called immediately before evaluation starts
         /// </summary>
         /// <param name="scope">The scope in which the code is executed</param>
         /// <param name="code">The code to be evaluated</param>
         /// <param name="bindingValues">The binding values - these are already added to the scope when called</param>
-        private static void OnEvaluationBegin(PyScope scope,
+        private void OnEvaluationBegin(PyScope scope,
```

Comment:
do these static events still exist in the base class?

Reply:
No...they do not exist anymore
SInce they are private and are note accessed by reflection anywhere....we do not need to keep it static